tests/e2e/specs/setup-mc/step-1-accounts.test.js

```diff
@@ -357,6 +357,96 @@ test.describe( 'Set up accounts', () => {
 				).toBeVisible();
 			} );
 		} );
+
+		test.describe( 'Claim Google Ads Account adsaccountclaim', () => {
+			test.beforeEach( async () => {
```

Comment:
If we can run these tests sequentially without needing to set up and refreshing the page each time it will speed up the tests. For more, see [these docs](https://playwright.dev/docs/api/class-test#test-before-all).

```suggestion
			test.beforeAll( async () => {
```